src/connectors/dynamic-widgets/connectDynamicWidgets.ts

```diff
@@ -18,6 +18,7 @@ export type DynamicWidgetsRenderState = {
 
 export type DynamicWidgetsConnectorParams = {
   widgets: Widget[];
+  fallbackWidget?(attribute: string): Widget;
```

Comment:
In React InstantSearch we pass it in an object:

```suggestion
  fallbackWidget?(params: { attribute: string }): Widget;
```